extensions/wikia/AdEngine/src/ad-engine.bridge.js

```diff
@@ -0,0 +1,55 @@
+import Context from 'ad-engine/src/services/context-service';
+import SlotService from 'ad-engine/src/services/slot-service';
+import TemplateService from 'ad-engine/src/services/template-service';
+import BigFancyAdBelow from 'ad-products/src/modules/templates/uap/big-fancy-ad-below';
+import UniversalAdPackage from 'ad-products/src/modules/templates/uap/universal-ad-package';
+import config from './context';
+
+Context.extend(config);
+let supportedTemplates = [ BigFancyAdBelow ];
+
+supportedTemplates.forEach((template) => {
+	TemplateService.register(template);
+});
+
+function init(slotRegistry) {
+	SlotService.getBySlotName = (id) => {
+		if (id) {
+			let slot = slotRegistry.get(id, 'DirectGpt');
+			return unifySlotInterface(slot);
+		}
+	};
+}
+
+function unifySlotInterface(slot) {
+	slot.getSlotName = () => slot.name;
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Convert this ES6 arrow function into a regular one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-js-rules%3AArrowFunctions)

[project-id]: wikia:app